cms/static/sass/elements/_modal-window.scss

```diff
@@ -514,17 +514,8 @@
       }
     }
 
-    .field-visibility-verification {
```

Comment:
Look @efischer19, I found more verification user partition changes I can delete! I'm removing some of the Studio code as part of my current work because I stumbled across it, and some of it I would need to modify if it stays in.